src/lib/i18n/locales/en/workflows.ts

```diff
@@ -105,6 +110,7 @@ export const Strings = {
   'signal-name-label': 'Signal name',
   'signal-payload-input-label': 'Data',
   'signal-payload-input-label-hint': '(only single JSON payload supported)',
+  'update-modal-title': 'Send an Update',
```

Comment:
Do we need this and `update` on line 56?

Reply:
Probably not, but I'd rather error on flexibility of separate translations in case the modal designs change.